docs/src/userguide/migrating_to_cy30.rst

```diff
@@ -173,6 +173,42 @@ The old behaviour can be restored with the
 :ref:`directive <compiler-directives>` ``c_api_binop_methods=True``.
 More details are given in :ref:`arithmetic_methods`.
 
+Exception values and ``noexcept``
+=================================
+
+``cdef`` functions that are not ``extern`` now propagate Python
+exceptions by default, where previously they needed to explicitly be
+declated with :ref:`exception value <error_return_values>` in order
+for them to do so. A new ``noexcept`` modifier can be used to declare
+``cdef`` functions that will not raise exceptions.
+
+In existing code, you should mainly look out for ``cdef`` functions
+that are declared without an exception value::
+
+  cdef int spam(int x):
+      pass
+
+If you left out the exception value by mistake, i.e., the function
+actually can raise Python exceptions, then the new behaviour will
+take care of this for you, and correctly propagate any exceptions.
+
+On the other hand, if you didn't declare an exception value because
+you know that the function cannot raise exceptions, the new behaviour
```

Comment:
```suggestion
On the other hand, if you didn't declare an exception value because
you want to avoid exceptions propagating out of this function, the new behaviour
```

I'm just rephrasing this because people might legitimately be using the "swallow exceptions" behaviour - it isn't required that the code can't raise exceptions.